source/docs/software/networktables/creating-sendable-classes.rst

```diff
@@ -0,0 +1,163 @@
+
+.. tabs::
+
+      .. code-tab:: c++
+
+        void MySubsystem::InitSendable(SendableBuilder& builder) {
+
+            builder.AddDoubleProperty(".intakeSpeed",
+                    [=]() { return
+                        myMotor.Get(); },
+                    [=](double value) {
+                        myMotor.Set(value);
+                });
+
+                Subsystem.InitSendable(builder);
+
+        }
+
+    .. code-tab:: java
+
+        public class MySubsystem extends Subsystem {
+
+            Spark spark = new Spark(1);
+            Solenoid solenoid = new Solenoid(0);
+
+            @Override
+            public void initSendable(SendableBuilder builder) {
+
+               builder.addDoubleProperty(".intakeSpeed", spark::get, (value) -> spark.set(value));
+
+               // this call sets up Command specific configuration, including calling setSmartDashboardType("Subsystem")
+               // to let Shuffleboard know to treat this as such.
+               super.initSendable(builder);
+            }
+
+        }
+
+Creating a new Sendable class with SendableBuilder
+--------------------------------------------------
+
+Creating an entirely new :code:`Sendable` class follows a similar pattern to the extending the implementation present in subclasses of WPI classes such as :code:`Command` and :code:`Subsystem` as presented above. The main difference is that when creating a new class, teams must explicitly define the Sendable's type. This allows teams to create custom wigets for Shuffleboard to display data. This example shows a data class which holds boolean and double values about an arm's state.
+
+.. tabs::
+
+    .. code-tab:: c++
+
+        TOOD
+
+    .. code-tab:: java
+
+        public class ArmState implement Sendable {
+
+            final double armAngle;
+            final boolean intakeOpen;
+            
+            public ArmState(double angle, boolean intakeOpen) {
+                this.armAngle = angle;
+                this.intakeOpen = intakeOpen;
+            }
+
+            public void initSendable(SendableBuilder builder) {
+                
+                builder.setSmartDashboardType("ArmState");
+                builder.addDoubleProperty(".angle", () -> armAngle, null);
```

Comment:
Properties starting with `.` are for metadata containing meta-information about the sendable that is not directly tied to the data that it is sending. Since the angle of an arm is _not_ meta-information, it should not use the `.` notation.

```suggestion
                builder.addDoubleProperty("Angle", () -> armAngle, null);
```